dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/DataOwnerControllerTest.kt

```diff
@@ -18,6 +18,12 @@ class DataOwnerControllerTest {
     private val apiAccessor = ApiAccessor()
     val jwtHelper = JwtAuthenticationHelper()
 
+    private fun createUnauthorizedUser() {
+        jwtHelper.authenticateApiCallsWithJwtForTechnicalUser(
+            TechnicalUser.entries.filter { it != TechnicalUser.Admin }.random(),
```

Comment:
Warum ein zufälliger Nutzer? Ich fände einen festen Nutzer besser, falls sich dann an den Nutzerrechten was ändern sollte und der Test fehlschlägt weiß man warum.
Oder gibt es einen Grund das für zufällige Nutzer testen zu wollen?

Reply:
Ich sehe keinen Grund für die Zufälligkeit. 
@Worben hast Du einen Einwand, das auf einen festen user (zB "reader") zu wechseln?

Reply:
Lets have a call about this

Reply:
Nach Absprache mit Torben, Idee war auch als nicht eingeloggter User zu testen.
Wir haben das jetzt mit einem leeren Token umgesetzt.

Reply:
Ok